site/public/phone-contacts.html

```diff
@@ -0,0 +1,193 @@
+
+		<link href="profiles/gsecms/themes/ausgovau_bootstrap/favicons/favicon-16x16.png" rel="shortcut icon" type="image/png" />
+		<link href="https://www.agls.gov.au/agls/terms/" rel="schema.AGLSTERMS" />
+		<meta content="width=device-width, initial-scale=1, maximum-scale=2, user-scalable=0" name="viewport" />
+		<meta content="true" name="HandheldFriendly" />
+		<meta content="width" name="MobileOptimized" />
+		<title>Coronavirus (COVID-19) - Helpful phone numbers</title>
+		<meta content="Q343DS0L1BSwg4xogPeW9R53riFpr_AuY8qsSD-rXl4" name="google-site-verification" />
+
+		<meta content="#00022c" name="msapplication-TileColor" />
+		<meta content="#00022C" name="theme-color" />
+		<meta content="yes" name="apple-mobile-web-app-capable" />
+		<meta content="black" name="apple-mobile-web-app-status-bar-style" />
+		<meta content="yes" name="mobile-web-app-capable" />
+		<meta content="IE=edge" http-equiv="X-UA-Compatible" />
+		<link href="https://fonts.googleapis.com/css?family=Montserrat:200,400,700|Roboto:300,400,500,700,900&display=swap" rel="stylesheet">
+
+		<link href="/homepage-resources/style.css" media="all" rel="stylesheet" type="text/css" />
+		<!-- <link href="/homepage-resources/main.css" media="all" rel="stylesheet" type="text/css" /> -->
+		<link href="/homepage-resources/essential.css" media="all" rel="stylesheet" type="text/css" />
+		<link href="/homepage-resources/auds.css" media="all" rel="stylesheet" type="text/css" />
+		<!--[if lt IE 9]>
+			<script src="/profiles/gsecms/themes/ausgovau_bootstrap/javascripts/vendor/html5shiv.min.js"></script>
+		<![endif]-->
+		<script src="profiles/gsecms/themes/ausgovau_bootstrap/javascripts/vendor/rem.min.js"></script>
+		<script src="profiles/gsecms/themes/ausgovau_bootstrap/javascripts/vendor/modernizr-n-respond.min.js"></script>
+	</head>
+
+	<body class="html au-grid front not-logged-in no-sidebars page-node">
+		<!-- Google Tag Manager (noscript) -->
+			<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PKZK44Z"
+				height="0" width="0" style="display: none; visibility: hidden;"></iframe>
+			</noscript>
+
+    <!-- End Google Tag Manager (noscript) -->
+
+    <nav class="au-skip-link" aria-label="skip links navigation">
+      <a class="au-skip-link__link" href="#main">Skip to main content</a>
+    </nav>
+    <header class="au-header au-header--dark" role="banner">
+      <div id="sitewide-alert" class="sitewide-alert is-active" aria-hidden="false">
+        <i class="info"></i>
+       <p> Find out if you need to get tested. &nbsp; <a href="https://www.healthdirect.gov.au/symptom-checker/tool?symptom=CORO">Go to Symptom Checker</a></p>
+      </div>
+        <div class="container-fluid">
+          <div class="logo">
+            <a href="/"><img class="au-header__brand-image" alt="Australian Government - Coat of Arms" src="/homepage-resources/resources/aus-gov-stacked-white.svg"></a>
+          </div>
```

Comment:
```suggestion
          <div class="logo">
          <center>
            <a href="/"><img class="au-header__brand-image" alt="Australian Government - Coat of Arms" src="/homepage-resources/resources/aus-gov-stacked-white.svg"></a>
            </center>
          </div>
```

Should the logo also be centred on this page?

Reply:
no - the content is left aligned. This is the same treatment as the other child page.